### PR TITLE
Mohammed_Alshowaikhat HW1 

### PR DESCRIPTION
**Pull request by Moe-Saeed**

### CODE REVIEW

File: Homeworks/HW1/mohammed_alshowaikhat/game_of_life.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
+def evolve(initial_state):
+
+    rows=len(initial_state)
+       cols=len(initial_state[0])
```
Comment (pusukuri) [inline Homeworks/HW1/mohammed_alshowaikhat/game_of_life.py:4]: Space should be proper to be able to compile. Please fix it.

**Review by pusukuri — CHANGES_REQUESTED** [review]
Indentation should be appropriate to be able to compile the code. Please fix it.

File: Homeworks/HW1/mohammed_alshowaikhat/game_of_life.py (hunk 1 above)
Comment (Moe-Saeed) [reply Homeworks/HW1/mohammed_alshowaikhat/game_of_life.py]: Changes are made, now it should work, thanks for your feedback

**Review by pusukuri — APPROVED** [review]
Good work...

**Pull request merged**